drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -79,20 +85,16 @@ static int bq274xx_control_reg_write(const struct device *dev,
 	return 0;
 }
 
-static int bq274xx_command_reg_write(const struct device *dev, uint8_t command,
-				     uint8_t data)
+static int bq274xx_command_reg_write8(const struct device *dev, uint8_t command,
+				     uint8_t value)
 {
 	const struct bq274xx_config *config = dev->config;
-	uint8_t i2c_data, reg_addr;
 	int status = 0;
 
-	reg_addr = command;
-	i2c_data = data;
-
-	status = i2c_reg_write_byte_dt(&config->i2c, reg_addr,
-				       i2c_data);
+	status = i2c_reg_write_byte_dt(&config->i2c, command,
+				       value);
 	if (status < 0) {
-		LOG_ERR("Failed to write into control register");
+		LOG_ERR("Failed to write into command register");
 		return -EIO;
 	}
```

Comment:
I would removing logging in these lower layer I2C functions as it doubles up with logging used in higher layer functions, see `bq274xx_gauge_configure()`.

Reply:
Also notable, this function is swallowing the I2C error code, it should really be returned to the calling function to use and/or log. The other lower level I2C functions should also follow this.